lib/server.coffee

```diff
@@ -126,7 +124,7 @@ WebApp.connectHandlers.use (req, res, next) ->
 
 		Spiderable.originalRequest = req
 		url 		= Spiderable._urlForPhantom Meteor.absoluteUrl(), req.url
-		hash 		= SHA256 url
+		hash 		= hashCode url
```

Comment:
SHA256 is heavy algorithm which targeted on security, but we don't need any security here. We only need unique value depended from `url`. I'm currently take a part in big project targeted on different kind of encryptions and encodings, and after I've get into it I realised how it was useless to use SHA256 to just generate content-dependable unique ids